examples/mnist.py

```diff
@@ -30,6 +32,11 @@ def main(save_to, num_epochs):
     cost = CategoricalCrossEntropy().apply(y.flatten(), probs)
     error_rate = MisclassficationRate().apply(y.flatten(), probs)
 
+    cg = ComputationGraph([cost])
+    W1, W2 = VariableFilter(roles=[VariableRole.WEIGHTS])(cg.variables)
+    cost = cost + .00005 * W1.norm(2) + .00005 * W2.norm(2)
```

Comment:
Actually L2 regularization means adding not norms but norms squared, so I guess you need `(W1 ** 2).sum()` instead.